docs/Glossary.md

```diff
@@ -230,7 +230,8 @@ A *route pattern* (or "path") is a string that describes a portion of a URL. Pat
 
   - `:paramName` – matches a URL segment up to the next `/`, `?`, or `#`. The matched string is called a [param](#params)
   - `()` – Wraps a portion of the URL that is optional
-  - `*` – Matches all characters (non-greedy) up to the next character in the pattern, or to the end of the URL if there is none, and creates a `splat` [param](#params)
+  - `*` – Matches all characters (non-greedy) up to the next character in the pattern, or to the end of the URL if there is none, and creates a `splat` [param](/docs/Glossary.md#params)
```

Comment:
Nit: Can you make those links not absolute? That means less work if we reorganize docs in the future, and when building them into a documentation website.